test/propagation.test.ts

```diff
@@ -54,13 +49,11 @@ describe('propagation', () => {
       assert.strictEqual(carrier['x-b3-spanid'], spanId);
       assert.strictEqual(carrier['x-b3-sampled'], '1');
       assert.strictEqual(carrier['traceparent'], `00-${traceId}-${spanId}-01`);
-      done();
```

Comment:
if for some reason this callback is not run then the asserts will never be run and we might get false positives

Reply:
By "callback" you mean `done`? If so the asserts in this test will be run regardless of us calling `done` or not. Do you have a code sample of a situation where the asserts would not be ran?

**Iff** the test function has a length of 1 (meaning we "ask" Mocha to make this an async test) `done` needs to be called though, if we don't add any arguments, Mocha assumes a synchronous test, which this is.

Reply:
By callback I mean the second argument to `context.with(spanContext, functionToRunInContext)`. I phrased it poorly.